pulsar-io/kafka/pom.xml

```diff
@@ -39,6 +39,12 @@
       <scope>provided</scope>
     </dependency>
 
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>pulsar-client-original</artifactId>
```

Comment:
Why do we need pulsar client explicitly for Kafka connector?

Reply:
@merlimat I believe this is related to the conversion between pulsar's schema and kafka's schema. So kafka connector need `pulsar-client-original` because all schema impls are in the package.

Reply:
yes, @freeznet is right. we need Schema Impl classes